src/components/SettingsPage.js

```diff
@@ -192,6 +193,7 @@ const SettingsPage = (props) => {
       >
         {children}
       </div>
+      <FooterV2 onBeforeInstall={() => {}} />
```

Comment:
@jedtan You can ignore that here. The user probably already has that info set in local storage, and if they don't, we'll say they'll be able switch to v4 just fine because they know it exists.